.github/workflows/lint.yml

```diff
@@ -19,3 +19,15 @@ jobs:
         run: |
           git fetch --no-tags --prune --depth=1 origin "${GITHUB_BASE_REF?}:${GITHUB_BASE_REF?}"
           ./build_tools/github_actions/lint_clang_format.sh -b "${GITHUB_BASE_REF}"
+
+  whitespace-checks:
+    # This job can only be run on pull_request since GITHUB_BASE_REF is only set on PR.
+    if: "github.event_name == 'pull_request'"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checking out repository
+        uses: actions/checkout@v2
+
+      - name: Running clang-format on changed source files
```

Comment:
"Running lint_whitespace_checks.sh on changed source files".

Reply:
"Running lint_whitespace_checks.sh checks on source files"

It doesn't only look at changed files currently. But that _shouldn't_ impact much. The check is lightweight.